commit 524295973653e269ee74ca609de14772cce7553b (refs/changes/39/1839/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-04-25T13:47:50-07:00 (2 years, 5 months ago)
    
    CMON-64 hermes should upload "cmon" service logs
    Reviewed by: Richard Kiene <richard.kiene@joyent.com>
    Approved by: Richard Kiene <richard.kiene@joyent.com>

diff --git a/etc/logsets.json b/etc/logsets.json
index f019b96..929fd32 100644
--- a/etc/logsets.json
+++ b/etc/logsets.json
@@ -213,6 +213,7 @@
       "binder",
       "ca",
       "cloudapi",
+      "cmon",
       "cnapi",
       "cns",
       "dhcpd",
